source/shared/smart-contracts/guides/setup-tools.rst

```diff
@@ -31,8 +31,16 @@ It can be used for :ref:`compiling<compile-module>` and
 
    Add links for testing and schemas.
 
-Cargo Concordium is distributed as part of the :ref:`Concordium software<downloads>` package.
-The tool should be placed in your PATH.
+Cargo Concordium is distributed as part of the :ref:`Concordium software<downloads>` package. Rename the downloaded file to ``cargo-concordium`` for MacOS/Linux, and ``cargo-concordium.exe`` for Windows.
+
+The tool should be placed in your PATH. This can be achieved by either:
+
+* Moving the executable to a folder that is already on your PATH, for example ``%HOMEPATH%\.cargo\bin\`` for Windows, and ``$HOME/.cargo/bin`` for MacOS/Linux.
+* Or, by adding its current location to your PATH environment.
+
+If you are running MacOS, you need to make the tool executable by running ``chmod +x path/to/cargo-concordium_1.0.0-0`` in a terminal. Make sure to provide the correct path to the downloaded tool. You also need to grant it permission to run in your  `Security & Privacy settings <https://support.apple.com/en-gb/guide/mac-help/mh40616/mac>`_ .
```

Comment:
```suggestion
If you are running MacOS, you need to make the tool executable by running ``chmod +x path/to/cargo-concordium` in a terminal. Make sure to provide the correct path to the downloaded tool. You also need to grant it permission to run in your  `Security & Privacy settings <https://support.apple.com/en-gb/guide/mac-help/mh40616/mac>`_ .
```

Would make sense to remove the version information here as well, if the user should have already renamed the file.